hikconnect/api.py

```diff
@@ -252,6 +252,22 @@ async def get_call_status(self, device_serial: str):
             "info": info,
         }
 
+    async def hangup(self, device_serial: str):
+        """
+        Send hangup request.
+
+        The `device_serial` parameter can be obtained from `get_devices()` and/or `get_cameras()`.
+        """
+        async with self.client.put(
+                f"{self.BASE_URL}/v3/devconfig/v1/call/{device_serial}/operation?cmdId=3&handler={self.client.FEATURE_CODE}"
```

Comment:
Is `&handler={self.client.FEATURE_CODE}` really needed? Can we test what happens if it is omitted?

Reply:
@tomasbedrich I can confirm that `&handler={self.client.FEATURE_CODE}` is not required and can be safely removed from the URL.

Reply:
Also, as [this post](https://community.home-assistant.io/t/ds-kd8003-ds-kv8113-ds-kv8213-ds-kv6113-ds-kv8413-and-integration-hikvision-hikconnect-video-intercom-doorbell/238535/1149), command `3` is cancel call and so I'll rename the function to cancelCall.

Reply:
Agree, just please use `cancel_call` (snake case) as per Python convention.

Reply:
@tomasbedrich I've updated the function to cancel_call. Please review let me know if you need anything.